presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/aggregation/TestGeometryUnionGeoAggregation.java

```diff
@@ -292,47 +298,66 @@
     public void testPoint(String testDescription, String expectedWkt, String... wkts)
     {
         assertAggregatedGeometries(testDescription, expectedWkt, wkts);
+        assertArrayAggAndStUnion(testDescription, expectedWkt, wkts);
     }
 
     @Test(dataProvider = "linestring")
     public void testLineString(String testDescription, String expectedWkt, String... wkts)
     {
         assertAggregatedGeometries(testDescription, expectedWkt, wkts);
+        assertArrayAggAndStUnion(testDescription, expectedWkt, wkts);
     }
 
     @Test(dataProvider = "polygon")
     public void testPolygon(String testDescription, String expectedWkt, String... wkts)
     {
         assertAggregatedGeometries(testDescription, expectedWkt, wkts);
+        assertArrayAggAndStUnion(testDescription, expectedWkt, wkts);
     }
 
     @Test(dataProvider = "multipoint")
     public void testMultiPoint(String testDescription, String expectedWkt, String... wkts)
     {
         assertAggregatedGeometries(testDescription, expectedWkt, wkts);
+        assertArrayAggAndStUnion(testDescription, expectedWkt, wkts);
     }
 
     @Test(dataProvider = "multilinestring")
     public void testMultiLineString(String testDescription, String expectedWkt, String... wkts)
     {
         assertAggregatedGeometries(testDescription, expectedWkt, wkts);
+        assertArrayAggAndStUnion(testDescription, expectedWkt, wkts);
     }
 
     @Test(dataProvider = "multipolygon")
     public void testMultiPolygon(String testDescription, String expectedWkt, String... wkts)
     {
         assertAggregatedGeometries(testDescription, expectedWkt, wkts);
+        assertArrayAggAndStUnion(testDescription, expectedWkt, wkts);
     }
 
     @Test(dataProvider = "geometrycollection")
     public void testGeometryCollection(String testDescription, String expectedWkt, String... wkts)
     {
         assertAggregatedGeometries(testDescription, expectedWkt, wkts);
+        assertArrayAggAndStUnion(testDescription, expectedWkt, wkts);
     }
 
     @Override
     protected String getFunctionName()
     {
         return "geometry_union_agg";
     }
+
+    private void assertArrayAggAndStUnion(String testDescription, String expectedWkt, String[] wkts)
```

Comment:
the implementation doesn't use `ST_Union` function, hence, the name is misleading; how about `assertGeometryUnion`?